gallia-prehistoire.csl

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-et-al="never" et-al-use-first="1" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <info>
+    <title>Gallia Préhistoire (French)</title>
+    <title-short>GP</title-short>
+    <id>http://www.zotero.org/styles/gallia-prehistoire</id>
+    <link href="http://www.zotero.org/styles/gallia-prehistoire" rel="self"/>
+    <link href="http://www.persee.fr/collection/galip" rel="documentation"/>
+    <link href="http://www.mae.u-paris10.fr/gallia-prehistoire/" rel="documentation"/>
+    <author>
+      <name>Thomas Perrin</name>
+      <email>tperrin@univ-tlse2.fr</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="anthropology"/>
+    <issn>0016-4127</issn>
+    <eissn>2109-9642</eissn>
+    <summary>Author-date style for Gallia Préhistoire - Préhistoire de la France dans son contexte européen</summary>
+    <updated>2017-11-14T12:51:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">dir.</term>
+      <term name="anonymous">anonyme</term>
+      <term name="anonymous" form="short">anon.</term>
+      <term name="accessed">consulté le</term>
+      <term name="no date">sans date</term>
+      <term name="no date" form="short">s.d.</term>
+      <term name="edition" form="short">éd.</term>
+      <term name="and">et</term>
+      <term name="forthcoming">à paraître</term>
+      <term name="in press">sous presse</term>
+      <term name="accessed">consulté le</term>
+      <term name="cited">op.&amp;#160;cit.</term>
+      <term name="page">p.</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if match="any" variable="author">
+        <names variable="author" font-variant="normal" delimiter=", ">
+          <name form="short" font-variant="normal" font-weight="normal" and="text" delimiter-precedes-last="never" et-al-min="3" et-al-subsequent-min="0" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
The "et-al" settings here are incorrect. "et-al-min" and "et-al-use-first" should always be defined together. "et-al-min" determines the minimum number of authors before "et al."-abbreviation kicks in, and "et-al-use-first" defines the number of authors to print before "et al.". "et-al-subsequent-min" and "et-al-subsequent-use-first" do the same for any additional time that a given item is cited.

In this case you have "et-al-min" and "et-al-subsequent-min", which is not a correct combo. The value of the "et-al" attributes should also always be higher than 0.